source/aggregation-pipeline-builder.txt

```diff
@@ -183,6 +183,39 @@ Open a Saved Pipeline
    To learn more, see 
    :ref:`viewing saved aggregations<saved-query-view>` for details.
 
+.. _set-maxtime-ms-agg:
+
+Set ``maxTimeMS`` for Aggregations Queries
+------------------------------------------
+
+Use the ``maxTimeMS`` parameter on the aggregation tab to put an 
```

Comment:
[suggestion/nit] suggesting a stronger verb such as "specify" for setting the parameter value